frontend/wallet/src/render/CodaProcess.re

```diff
@@ -120,9 +47,14 @@ let useHook = () => {
     Hooks.Reducer.useReducer(reduce, {State.args: [], mode: Stable});
 
   let () =
-    React.useEffect0(() => {
+    React.useLayoutEffect0(() => {
```

Comment:
Why are you switching to `useLayoutEffect` here? As far as I can tell, you only want to use this when the logic here affects the DOM in some way.

Reply:
Shoot I thought I pushed a revert to this already